curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.english.md

```diff
@@ -26,7 +26,7 @@ tests:
   - text: <code>myData</code> should be equal to <code>8</code>.
     testString: assert(myData === 8, '<code>myData</code> should be equal to <code>8</code>.');
   - text: You should be using bracket notation to read the correct value from <code>myArray</code>.
-    testString: assert(/myArray\[2\]\[1\]/g.test(code) && !/myData\s*=\s*(?:.*[-+*/%]|\d)/g.test(code), 'You should be using bracket notation to read the correct value from <code>myArray</code>.');
+    testString: assert(/myData=myArray\[2\]\[1\]/.test(code.replace(/\s/g, 'You should be using bracket notation to read the correct value from <code>myArray</code>.'));
```

Comment:
You did not copy/paste the whole assert as referenced in the issue comment.  Technically, you do not need the assert message argument due to some recent changes in the testing code.
```suggestion
    testString: assert(/myData=myArray\[2\]\[1\]/.test(code.replace(/\s/g, ''));
```